databases/week1/index.sql

```diff
@@ -0,0 +1,57 @@
+/* Find out how many tasks are in the task table */
+SELECT COUNT(id)
+FROM task;
+
+/* Find out how many tasks in the task table do not have a valid due date */
+SELECT COUNT(id)
+FROM task
+WHERE `due_date` IS NULL;
+
+/* Find all the tasks that are marked as done */
+SELECT title
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name = "Done";
+
+/* Find all the tasks that are not marked as done */
+SELECT title
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name != "Done";
+
+/* Get all the tasks, sorted with the most recently created first */
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+/* Get the single most recently created task */
+SELECT *
+FROM task
+ORDER BY created DESC
+LIMIT 1;
+
+/* Get the title and due date of all tasks where the title or description contains database */
+SELECT task.title, task.due_date 
```

Comment:
the table name is not necessary here, it is not wrong though.
```suggestion
SELECT title, due_date 
```